functional_tests/scylla_operator/test_functional.py

```diff
@@ -86,6 +104,7 @@ def test_drain_and_replace_node_kubernetes(db_cluster):
     target_node.wait_till_k8s_pod_get_uid(ignore_uid=old_uid)
     target_node.wait_for_pod_readiness()
     db_cluster.wait_for_pods_readiness(pods_to_wait=1, total_pods=len(db_cluster.nodes))
+    target_node.refresh_ip_address()
```

Comment:
In the "test_cassandra_rackdc" we start using "nodetool status" command for checking nodes statuses expecting "UN" there. 
And if we run these 2 not yet updated tests with the drain logic before rackdc test then it will fail.
The cause for failure will be different IP addresses in "db_cluster.nodes" and real Scylla nodes registered in cluster.
So, in other words, the existing bug in 2 drain tests start influencing the "rackdc" one by the fact of adding "nodetool status" verification.